data/mods/roulettemons/random-teams.ts

```diff
@@ -1,117 +0,0 @@
-import {PRNG, PRNGSeed} from '../../../sim/prng';
-import RandomTeams from '../../random-teams';
-
-export class RandomRoulettemonsTeams extends RandomTeams {
```

Comment:
Notice how Roulettemons extends the existing `RandomTeams` class and only overrides the relevant methods methods? You can probably do something similar for Joltemons.

Reply:
[https://github.com/scoopapa/DH/blob/Yoshiblaze-jolterandbats2/data/mods/joltemons/random-teams.ts](url) would this work?